docs/_docs/windows.md

```diff
@@ -8,7 +8,56 @@ Jekyll with the proper tweaks. If you are using Windows 10 Anniversary Update,
 the easiest way to run Jekyll is to use the new [Bash on Ubuntu on Windows](https://msdn.microsoft.com/en-us/commandline/wsl/about?f=255&MSPPError=-2147217396).
 For older installations, this page aims to collect some of the general knowledge and lessons that have been unearthed by Windows users.
 
-## Installation
+## Installation via Bash on Windows 10
+
+*Please note:* You must have [Bash on Ubuntu on Windows](https://msdn.microsoft.com/en-us/commandline/wsl/about?f=255&MSPPError=-2147217396) enabled.
+
+First lets make sure all our packages / repositories are up to date. Open a new Command Prompt instance, and type the following:
+
+```
+bash
+```
+Your Command Prompt instance should now be a Bash instance. Now we must update our repo lists and packages.
+
+```
+sudo apt-get update -y && sudo apt-get upgrade -y
+```
+
+Now we can install Ruby. First we need to add the repository list from BrightBox.
+
+```
+sudo apt-add-repository ppa:brightbox/ruby-ng
+sudo apt-get update
+sudo apt-get install ruby2.3 ruby2.3-dev build-essentials
+```
+Next lets update our Ruby gems:
+
+```
+sudo gem update
+
+```
+
+Now all that is left to do is install Jekyll.
+
+```
+sudo gem install jekyll bundler
+
+```
+
+You can test by running:
+
+```
+jekyll new my_project
+```
+
+
+**And thats it!**
+If you cd into the folder, you can make sure time management is working by opening your "_posts"_ folder. You should see a markdown file with the current date listed.
```

Comment:
`"_posts"_` is probably the wrong markup for this? consider using backticks here